src/Storefront/Theme/Controller/ThemeController.php

```diff
@@ -45,11 +57,39 @@ public function configuration(string $themeId, Context $context): JsonResponse
         return new JsonResponse($themeConfiguration);
     }
 
+    /**
+     * Updates the configuration of a specific theme.
+     *
+     * This endpoint allows updating theme configuration values.
+     * If the reset parameter is provided, the theme will be reset
+     * to its default configuration before applying any new values.
+     * If the validate parameter is provided, the theme config will be validated before updating the theme.
+     * If the sanitize parameter is provided, the theme config will be sanitized during validation.
+     *
+     * @param string $themeId The unique identifier of the theme to update
+     * @param Request $request The HTTP request containing configuration data
+     * @param Context $context The application context containing user and system information
+     *
+     * @return JsonResponse Returns an empty JSON response on successful update
+     */
     #[Route(path: '/api/_action/theme/{themeId}', name: 'api.action.theme.update', methods: ['PATCH'])]
     public function updateTheme(string $themeId, Request $request, Context $context): JsonResponse
     {
         $config = $request->request->all('config');
 
+        $validateConfig = $request->query->getBoolean('validate', false);
+
+        // Validate the theme config before updating the theme.
+        if ($validateConfig) {
+            $config = $this->themeService->validateThemeConfig(
+                $themeId,
+                $config,
+                $context,
+                $this->customAllowedRegex,
+                $request->request->getBoolean('sanitize', false)
```

Comment:
```suggestion
                $request->query->getBoolean('sanitize', false)
```

at least it is also listed as parameter in the schema JSON file